### PR TITLE
Vhbb heppy74 x add metsv2

### PR DESCRIPTION
**Pull request by degrutto**
fix previous pull requests (now  starting from 745)

added

tkMET
rawmet
type1p2met
puppiMET
mentNoPU
mht30

just the pt....

### CODE REVIEW

File: VHbbAnalysis/Heppy/python/VHbbAnalyzer.py [hunk 1]
```diff
@@ -139,6 +139,15 @@ def doFakeMET(self,event):
 		event.fakeMET=event.met.p4() + event.V
                 event.fakeMET.sumet = event.met.sumEt() - event.V.pt()
 
+    def doHtMhtJets30(self,event):
+        ## with Central Jets
+        objects30 = [ j for j in event.cleanJets if j.pt() > 30 ] + event.selectedLeptons
+        event.htJet30 = sum([x.pt() for x in objects30])
+        event.mhtJet30vec = ROOT.reco.Particle.LorentzVector(-1.*(sum([x.px() for x in objects30])) , -1.*(sum([x.py() for x in objects30])), 0, 0 )
```
Comment (arizzi) [inline VHbbAnalysis/Heppy/python/VHbbAnalyzer.py]: one could even just some the p4() passing a 0,0,0,0 Vector as start of the sum

File: VHbbAnalysis/Heppy/test/vhbb.py [hunk 2]
```diff
@@ -46,9 +47,23 @@
...
+		 NTupleVariable("metPuppi_pt",  lambda ev : ev.metPuppi.pt(), help="met from Puppi"),
+		 NTupleVariable("metPuppi_rawpt",  lambda ev : ev.metPuppi.uncorrectedPt(), help="raw met from Puppi"),
+		 NTupleVariable("metType1p2_pt",  lambda ev : ev.met.shiftedPt(12,2), help="type1type2met from Puppi"),
+		 NTupleVariable("metNoPU_pt",  lambda ev : ev.metNoPU.pt(), help="PFnoPU E_{T}^{miss}"),
+		 NTupleVariable("tkMet_pt",  lambda ev : ev.tkMet.pt(), help="E_{T}^{miss} from tracks"),
+		 NTupleVariable("tkMetPVchs_pt",  lambda ev : ev.tkMetPVchs.pt(), help="E_{T}^{miss} from tracks"),
 	],
 	globalObjects = {
           "met"    : NTupleObject("met",     metType, help="PF E_{T}^{miss}, after default type 1 corrections"),
+#          "metPuppi"    : NTupleObject("metPuppi",     metType, help="PF E_{T}^{miss} from Puppi, after default type 1 corrections"),
```
Comment (arizzi) [inline VHbbAnalysis/Heppy/test/vhbb.py]: can we just remove those unless they are needed?

File: VHbbAnalysis/Heppy/test/vhbb.py [hunk 3]
```diff
@@ -163,13 +178,25 @@
...
+
 
 from PhysicsTools.Heppy.analyzers.core.PileUpAnalyzer import PileUpAnalyzer
 PUAna = PileUpAnalyzer.defaultConfig
 
 from VHbbAnalysis.Heppy.VHbbAnalyzer import VHbbAnalyzer
 JetAna.jetPt = 15
-JetAna.doQG=True
+#JetAna.doQG=True
+JetAna.doQG=False
```
Comment (arizzi) [inline VHbbAnalysis/Heppy/test/vhbb.py]: why??

Comment (degrutto) [inline VHbbAnalysis/Heppy/test/vhbb.py]: sorry that is a left out (to do a faster test),
I will correct it

File: VHbbAnalysis/Heppy/test/vhbb.py (hunk 2 above)
Comment (degrutto) [inline VHbbAnalysis/Heppy/test/vhbb.py]: ok

File: VHbbAnalysis/Heppy/python/VHbbAnalyzer.py (hunk 1 above)
Comment (degrutto) [inline VHbbAnalysis/Heppy/python/VHbbAnalyzer.py]: yes, but it is really the same,
if you do not insist I would keep it like this

Comment (arizzi) [inline VHbbAnalysis/Heppy/python/VHbbAnalyzer.py]: definitelly I do not insist! it was just a comment

**Pull request merged**